### PR TITLE
fix: event card should show upcoming occurrences only, no matter the enrolment time

### PR DESCRIPTION
**Pull request by nikomakela**
PT-1310. NOTE: this needs https://github.com/City-of-Helsinki/palvelutarjotin/pull/229 to be merged.

### CODE REVIEW

File: src/domain/events/__tests__/EventCard.test.tsx [hunk 1]
```diff
@@ -190,3 +192,20 @@ it('renders multiday occurrence time correctly', async () => {
 
   expect(container).toMatchSnapshot();
 });
+
+it('renders date as tomorrow and ignores enrolment days', async () => {
+  Date.now = jest.fn(() => new Date(Date.UTC(2020, 8, 19)).valueOf());
```
Comment (Jokinen) [inline src/domain/events/__tests__/EventCard.test.tsx:197]: Could be that timer mocks would be a better fit here: https://jestjs.io/docs/timer-mocks

Comment (Sartonon) [reply src/domain/events/__tests__/EventCard.test.tsx:197]: jest-date-mock has been used in kultus

Comment (nikomakela) [reply src/domain/events/__tests__/EventCard.test.tsx:197]: I tried it with timer-mocks, but couldn't get it to work. I thought this is good enough.

Comment (Sartonon) [reply src/domain/events/__tests__/EventCard.test.tsx:197]: `advanceTo` was not working?

Comment (nikomakela) [reply src/domain/events/__tests__/EventCard.test.tsx:197]: `advanceTo` works! 👍  Changed the implementation to that. 

Had to look inside the code: https://github.com/hustcc/jest-date-mock/blob/master/src/mockDate.js. Sure it's easier to use, but I would say that it's more overwhelming and didn't give anything more here.

**Comment by Jokinen** [comment]
Seems like this PR also needs these test fixes: https://github.com/City-of-Helsinki/palvelutarjotin-ui/pull/189

**Review by Sartonon — APPROVED** [review]
Looks good!

**Review by Jokinen — APPROVED** [review]

**Pull request merged**